docs/developer-docs/multi-chain/faq/user-faq.mdx

```diff
@@ -12,11 +12,30 @@ The following questions and answers are from the POV of a user using consumer ap
 
 For questions from the POV of a developer who wants to know things such as "how does a canister send ckBTC?" or how to write code for smart contracts to hold, send, or receive, please see the other FAQs.
 
+## General multi-chain questions
+
+### What does it mean that the Internet Computer is multi-chain?
+
+The Internet Computer (ICP) is described as multi-chain because it can interact 
+directly with other blockchains without the need for trusted centralized bridges.
+This means smart contracts on the Internet Computer can call into different blockchains.
+This is made possible through the use of advanced cryptographic mechanisms, collectively known as chain-key cryptography.
+
+### What is the purpose of the Internet Computer being having multi-chain smart contracts that can call into other blockchains?
+
+Three reasons:
+
+1. ICP canister smart contracts have a lot of functionality that other blockchain smart contracts do not, so this allows developers of other blockchains have access to ICP smart contracts while still writing contracts for some other token or blockchain.
```

Comment:
... this allows ... have access to – this reads weird to me.

Reply:
fixed